### ISSUE TITLE
Pseudo elements ::before and ::after don't work when using object syntax

### ISSUE THREAD

**Issue by adamsoffer** [styled-components/styled-components#2197]
### System:
 - OS: macOS 10.14
 - CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
 - Memory: 69.29 MB / 8.00 GB
 - Shell: 5.3 - /bin/zsh
### Binaries:
 - Node: 8.12.0 - ~/.nvm/versions/node/v8.12.0/bin/node
 - Yarn: 1.9.4 - /usr/local/bin/yarn
 - npm: 6.4.1 - ~/.nvm/versions/node/v8.12.0/bin/npm
 - Watchman: 4.9.0 - /usr/local/bin/watchman
### npmPackages:
 - styled-components: ^4.0.3 => 4.0.3 

## Steps to reproduce
Create a styled component in object syntax with a pseudo selector. ie:
```
const StyledDiv = styled.div({
  '::before': {
    content: '🚀'
  }
})
```

## Expected Behavior
We should see a pseudo element inside the div.

## Actual Behavior
A pseudo element doesn't get added to the div

**Comment by mxstbr** [comment]
This should work I think

```JS
const StyledDiv = styled.div({
  '&::before': {
    content: '🚀'
  }
})
```

**Comment by abolajibisiriyu** [comment]
@mxstbr that approach doesn't work. However i found a workaround 
```
const Box = styled(
  styled.div({
    background: "yellow",
    height: "50px",
    width: "50px",
    "&::after": {
      content: "after"
    }
  })
)`
  &::after {
    content: "after";
  }
`;
```
i hope it helps 🙂

**Comment by probablyup** [comment]
Does actually work, there's just a nuance around how the `content` property needs to be set: https://codesandbox.io/s/lxm7q494kz

**Comment by probablyup** [comment]
We could probably add a note about this to the docs

**Comment by adamsoffer** [comment]
Ah yup the extra quotations did the trick. Thanks @probablyup !

**Comment by revelt** [comment]
TLDR — **wrap value with extra quotes**:

BAD:

```js
&&:after {
    content: ${(props) => `${props.cta ? props.cta : 'Click'}`};
}
```

GOOD:

```js
&&:after {
    content: ${(props) => `"${props.cta ? props.cta : 'Click'}"`};
}
```

**Comment by gr8temi** [comment]
> TLDR — **wrap value with extra quotes**:
> 
> BAD:
> 
> ```js
> &&:after {
>     content: ${(props) => `${props.cta ? props.cta : 'Click'}`};
> }
> ```
> 
> GOOD:
> 
> ```js
> &&:after {
>     content: ${(props) => `"${props.cta ? props.cta : 'Click'}"`};
> }
> ```

Can you explain how this works instead of the returned string literal

**Comment by revelt** [comment]
@gr8temi My hypothesis is there's some sort of `eval` going on, the "string" statement (without quotes) is statement, but with quotes it becomes string. I guess maintainers would have more of a clue... Or you can delve into the source code...

**Comment by kitten** [comment]
There's a pretty simple explanation; The `content` property in CSS expects a CSS strong which needs to be quoted.

When we just define a string in JS in an interpolation or as an object value, then it's just inserted into CSS, so this happens:

```css
/* This happens and is wrong: */
content: Some Text;
/* But this is what we want: */
content: "Some Text";
```

So in JS we need to use quotes. We could do this with `JSON.stringify` or by using explicit quotes inside our JS string.

I'll go ahead lock this issue, since it's resolved and two years old, so people won't have to expect notifications from old issues.
As always, new questions can be directed to Spectrum or StackOverflow
